src/Teams/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -230,14 +230,17 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("TeamName")
                         .IsRequired()
                         .HasColumnName("TeamName")
-                        .HasColumnType("nvarchar(max)");
+                        .HasColumnType("nvarchar(450)");
```

Comment:
why changed?

Reply:
I changed it in order to keep consistency. 
The actual data type in the TeamName table of the TeamCapacity database is nvarchar(450):
* [TeamName]  NVARCHAR (450) NOT NULL,
Should I revert the .HascolumnType back to nvarchar(max) instead?